pkg/prometheus/promcfg.go

```diff
@@ -215,6 +216,31 @@ func buildExternalLabels(p *v1.Prometheus) yaml.MapSlice {
 	return stringMapToMapSlice(m)
 }
 
+// ValidateConfigInputs validates if inputs passed are valid
+func (cg *ConfigGenerator) ValidateConfigInputs(p *v1.Prometheus) error {
+	logger := log.With(cg.logger, "Validating config inputs", "EnforcedBodySizeLimit")
```

Comment:
Since the logger isn't used directly in this func, no need to create a new contextual logger. And the "key" parameter should be snake case instead of a full sentence.